scrapy_zyte_smartproxy/middleware.py

```diff
@@ -220,9 +220,9 @@ def process_request(self, request, spider):
             if self.job_id:
                 job_header = 'Zyte-JobId' if targets_zyte_api else 'X-Crawlera-JobId'
                 request.headers[job_header] = self.job_id
-            if not targets_zyte_api:
-                from scrapy_zyte_smartproxy import __version__
-                request.headers['X-Crawlera-Client'] = 'scrapy-zyte-smartproxy/%s' % __version__
+            user_agent_header = "Zyte-Client" if targets_zyte_api else "X-Crawlera-Client"
```

Comment:
Is using `X-` obsolete?

Reply:
[Yes, it stopped being a best practice some time ago](https://datatracker.ietf.org/doc/html/rfc6648).